src/Login/Login.js

```diff
@@ -1,25 +1,47 @@
 import React from 'react'
-import { Redirect, Link } from 'react-router-dom';
+import { withRouter, Link, Redirect } from 'react-router-dom';
 import { Component } from 'react';
 
-import { postData } from './apiFetch.js'
-import App from './App.js';
+import { postData } from '../apiFetch.js'
+import App from '../App/App.js';
 
 
 class Login extends Component {
-  constructor() {
-    super()
+  constructor(props) {
+    super(props)
     this.state = {
       email: '',
       password: '',
       loggedIn: false,
     }
   }
 
+  clearInputs = () => {
+    this.setState({email: '', password: ''})
+  }
+
+  updateValue = (event) => {
+    this.setState({ [event.target.name]: event.target.value })
+  }
+
+  loginHandler = (event) => {
+
+    event.preventDefault();
+    postData(this.state.email, this.state.password)
+    .then(response => console.log(response))
+    .then(() => this.clearInputs())
+    .then(this.setState({loggedIn: true}))
+    // .then(this.resetLoggedInState())
+    .catch(error => console.log('Not fetching user data'))
```

Comment:
Same thing with this error - instead of console logging it, what else can you do with this information?